tools/ansible-pr/generate_template.rb

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env ruby
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'optparse'
+require_relative '../../compile/core'
+require 'octokit'
+require 'time'
+
+# Class to handle Compile include to avoid Rubocop issues.
+class CompileClass
+  include Compile::Core
+end
+
+@options = {
+  directory: "#{Dir.pwd}/build/ansible/",
+  dry_run: false,
+  # Get all PRs from one day ago.
+  date: (Time.now - (3600 * 24)).iso8601
+}
+
+TEMPLATE = "
+<%= ctx[:commit_msg] %>
+
+##### SUMMARY
+<%= ctx[:commit_msg] %>
+<% if ctx[:prs] && !ctx[:prs].empty?-%>
+This PR is the result of the following Magic Modules PRs:
+<% ctx[:prs].each do |pr| -%>
+  - [<%= pr[:title] -%>](<%= pr[:url] -%>)
+<% end -%>
+<% end -%>
+
+##### ISSUE TYPE
+<% if ctx[:new] -%>
+- New Module Pull Request
+<% else -%>
+- Bugfix Pull Request
+<% end -%>
+
+##### ADDITIONAL INFORMATION
+```
+This was generated by [Magic Modules](https://github.com/googlecloudplatform/magic-modules)
+```
+".freeze
+
+@options = {
+  new: false,
+  mod_name: '',
+  date: (Time.now - (3600 * 24)).iso8601,
+  creds: nil
+}
+
+def all_pr_messages
+  # Limit of 50 bug fixes will go into a PR (should never hit this limit)
+  client = Octokit::Client.new(access_token: @options[:creds], per_page: 50)
+  client.list_issues('googlecloudplatform/magic-modules',
```

Comment:
Have you tested this?  I thought repo names were case sensitive and that the labels would be a list instead of a string.

Reply:
I've had zero issues. Works like a dream.